.eslintrc.js

```diff
@@ -0,0 +1,213 @@
+module.exports = {
```

Comment:
any reason to hardcode everything here, instead of basing it on a popular common shared config?

Reply:
just copying my boilerplate opinionated config.  I'm fine with changing

Reply:
i'd personally prefer something minimal, and static, and largely based on something commonly used.

Reply:
Or just don't bother. There's [a bunch of scripts](https://github.com/tc39/ecma262/tree/main/scripts) in ecma262 and those aren't linted. Since they're almost never touched it doesn't really matter.

Reply:
That's also a viable choice.

Reply:
🔥 nuked the rules block